docs/cli.rst

```diff
@@ -0,0 +1,45 @@
+Command Line Interface
+======================
+
+Nameko ships with a command line interface to make hosting and interacting with services as easy as possible.
+
+Running a Service
+-----------------
+
+::
+
+    $ nameko run [<module>[:<ServiceClass>]]
+
+Discover and run a service class.
```

Comment:
Yeah there's a whole missing section on deployment or possibly a "service development guide" describing some of the patterns we use in platform (that aren't enforced but we think are sensible)